EIPS/eip-5131.md

```diff
@@ -110,6 +111,7 @@ In solidity, the validation signature would be:
 function validate(address ensRegistry, bytes calldata senderENS, address mainAddress, string[] memory mainENSParts) view returns(bool);
 ```
 
+This can be an internal validate function to the smart contract, that is used in conjunction with specs like EIP-1721: Standard Signature Validation Method for Contracts and Merkle Proof Verification.
```

Comment:
```suggestion
This can be an internal validate function to the smart contract, that is used in conjunction with specs like [EIP-1721](./eip-1721.md): Standard Signature Validation Method for Contracts and Merkle Proof Verification.
```

Should be linked

Reply:
Thanks.  Have made edits.